cheri/cheri_prelude_common.sail

```diff
@@ -520,3 +520,6 @@ function dump_cp2_state () = {
   print(concat_str("DEBUG CAP HWREG 31", capToString(capRegToCapStruct(EPCC))));
 
 }
+
+val bitvector_concat = {ocaml: "append", lem: "concat_vec"} : forall ('n : Int) ('m : Int).
```

Comment:
This is already defined in vector_dec standard library header (not documented, sorry) and should also be possible to use @ as above.